templates/cosmic-settings.tera

```diff
@@ -0,0 +1,135 @@
+---
+name: "Catppuccin"
+roundness: "round"
+window_hint_color: ""
+bg_alpha: 1.0
+frosted: false
+outer_gap_size: 0
+inner_gap_size: 8
+active_hint_size: 3
+whiskers:
+    version: 2.5.1
+    matrix:
+    - flavor
+    - accent
+    filename: "themes/cosmic-settings/{{ name }}-{{ flavor.identifier | capitalize }}-{{ accent | capitalize }}+{{ roundness }}.ron"
```

Comment:
I'd appreciate if we could lowercase these generated files since it would match ctp/cursors, ctp/gtk. I'd like for lowercase files to be the standard going forward in the organisation.

Also removes the frontmatter variable as discussed.

```suggestion
roundness: "round"
window_hint_color: ""
bg_alpha: 1.0
frosted: false
outer_gap_size: 0
inner_gap_size: 8
active_hint_size: 3
whiskers:
    version: 2.5.1
    matrix:
    - flavor
    - accent
    filename: "themes/cosmic-settings/catppuccin-{{ flavor.identifier }}-{{ accent }}+{{ roundness }}.ron"
```

Reply:
Should the names in the theme name (inside the generated files, not the filename) be lowercase or uppercase?

Reply:
AFAIK the settings theme name isn't displayed anywhere in the interface, but given the field exists, it may at some point. The terminal theme is displayed in the interface though.